lib/galaxy/datatypes/converters/interval_to_tabix_converter.xml

```diff
@@ -1,18 +1,26 @@
-<tool id="CONVERTER_interval_to_tabix_0" name="Convert Interval to tabix" version="1.0.1" hidden="true" profile="16.04">
+<tool id="CONVERTER_interval_to_tabix_0" name="Convert Interval to tabix" version="1.0.2" hidden="true" profile="16.04">
     <!-- <description>__NOT_USED_CURRENTLY_FOR_CONVERTERS__</description> -->
     <requirements>
-        <requirement type="package" version="0.15.4">pysam</requirement>
+        <requirement type="package" version="1.14">htslib</requirement>
     </requirements>
     <command><![CDATA[
-        python '$__tool_directory__/interval_to_tabix_converter.py'
+        ln -s '$bgzip' input.bgz &&
+        tabix 
         #if $input1.ext in ['bed', 'gff', 'vcf']
-            -P $input1.ext
+            -p $input1.ext
```

Comment:
```suggestion
        #if $input1.is_of_type('bed')
            -p bed
        #elif $input1.is_of_type('vcf')
            -p vcf
        #elif $input1.is_of_type('gff')
            -p gff
```
(You need to remove the line above)

Reply:
Thanks for the suggestion. Will apply it. 

For making suggestions in github you can also select multiple lines.

Reply:
I don't manage as the first line has not been modified...